pkg/controller/clusterstate/api/node_info.go

```diff
@@ -75,11 +78,12 @@ func NewNodeInfo(node *v1.Node) *NodeInfo {
 			Allocatable: EmptyResource(),
 			Capability:  EmptyResource(),
 
-			Labels: make(map[string]string),
+			Labels:        make(map[string]string),
 			Unschedulable: false,
-			Taints: []v1.Taint{},
+			Taints:        []v1.Taint{},
 
-			Tasks: make(map[TaskID]*TaskInfo),
+			Tasks:   make(map[TaskID]*TaskInfo),
+			IsReady: "True",
```

Comment:
Recommend assignment value to by of the type `v1.ConditionStatus.ConditionTrue` to be consistent with the type defined above `v1.ConditionStatus``

Reply:
Done